File: app/src/main/java/co/cdmunoz/snaphelperexample/CandyView.java
package co.cdmunoz.snaphelperexample;

import co.cdmunoz.snaphelperexample.data.model.Candy;
import java.util.List;

public interface CandyView {

  void showLoading();
  void hideLoading();
  void showCandies(List<Candy> candies);
  void showError();

}

File: app/src/main/java/co/cdmunoz/snaphelperexample/CandyView.kt
package co.cdmunoz.snaphelperexample

import co.cdmunoz.snaphelperexample.data.model.Candy

interface CandyView {

  fun showLoading()
  fun hideLoading()
  fun showCandies(candies: List<Candy>)
  fun showError()

}
